.ci/static-checks.sh

```diff
@@ -239,6 +251,9 @@ static_check_commits()
 	# for main branch
 	(cd "${tests_repo_dir}" && make checkcommits && git remote set-branches origin 'main' && git fetch -v)
 
+	command -v checkcommits &>/dev/null || \
+		die 'checkcommits command not found. Ensure that "$GOPATH/bin" is in your $PATH.'
```

Comment:
Nit: Alternatively, you could create a function that accepts the command name and calls die specifying `$cmd` if not found to avoid the duplication in all these calls.